travis.sh

```diff
@@ -1,12 +1,17 @@
 #!/bin/sh
 set -e #break script on non-zero exitcode from any command
+set -x #display command being executed
```

Comment:
Actually let's leave it there. It would make it consistent with AppVeyor and provide value :+1: